tests/test.py

```diff
@@ -76,22 +82,19 @@ def test_stopping_vm(self):
         cmd = self.bin_dir + "minishift stop"
         self.execute_test({ 'cmd': cmd })
 
-        # Check VM status status
+    def test_status_after_stop(self):
         cmd = self.bin_dir + "minishift status"
-        self.log.info("Executing command : %s" % cmd)
-        process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
-        output = process.communicate()[0]
-        self.assertEqual(0, process.returncode)
-        self.assertEqual('Stopped', output.rstrip())
+        output = self.execute_test({ 'cmd': cmd })
+        self.assertEqual(output.rstrip(), 'Stopped')
 
-    def test_swapspace(self):
+    def test_restart(self):
```

Comment:
No need. The same as for the start, when the process returns a non-zero exit code/stderr, it will fail as test.